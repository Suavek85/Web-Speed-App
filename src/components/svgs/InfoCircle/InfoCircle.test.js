import React from 'react'
import { shallow } from 'enzyme'
import InfoCircle from './InfoCircle'

describe('InfoCircle snapshot', () => {

  it('should match the snapshot', () => {
    const wrapper = shallow(<InfoCircle />)
    expect(wrapper).toMatchSnapshot()
  })

})