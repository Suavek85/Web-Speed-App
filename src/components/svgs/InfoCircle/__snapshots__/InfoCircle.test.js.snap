// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InfoCircle snapshot should match the snapshot 1`] = `
<svg
  data-tip={true}
  height="18px"
  viewBox="0 0 40 40"
  width="18px"
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    d="M20,38.5C9.799,38.5,1.5,30.201,1.5,20S9.799,1.5,20,1.5S38.5,9.799,38.5,20S30.201,38.5,20,38.5z"
    fill="#98ccfd"
  />
  <path
    d="M20,2c9.925,0,18,8.075,18,18s-8.075,18-18,18S2,29.925,2,20S10.075,2,20,2 M20,1 C9.507,1,1,9.507,1,20s8.507,19,19,19s19-8.507,19-19S30.493,1,20,1L20,1z"
    fill="#4788c7"
  />
  <path
    d="M20 10A2 2 0 1 0 20 14A2 2 0 1 0 20 10Z"
    fill="#fff"
  />
  <g>
    <path
      d="M22 29L22 17 17 17 17 18 18 18 18 29 17 29 17 30 23 30 23 29z"
      fill="#fff"
    />
  </g>
</svg>
`;
